staging/src/k8s.io/apiextensions-apiserver/test/integration/fixtures/server.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
2018

Reply:
This is a rename that git does not detect.